Added blog entry (& myself as an author) for tomorrow's announcement (#722)
gitpod-io/website: src/blog/opensource.md

---
date: Tue, 25 Aug 2020 17:00:00 UTC
author: svenefftinge, JohannesLandgraf
subtitle:
title: Gitpod is now Open Source 🎉
image: /confetti.jpg
teaserImage: /confetti.jpg
---

As of today Gitpod is open source under the AGPL license at [github.com/gitpod-io/core](https://github.com/gitpod-io/core). We want to allow the community to participate in the development of Gitpod, provide more transparency and make it even easier for developers to use and integrate Gitpod in their workflows.

> For those of you who know us, this probably does not come as a big surprise. Working in open source is in our DNA and everything we’ve created over the past 10 years, including [Theia](https://github.com/eclipse-theia/theia), [Xtext](https://github.com/eclipse/xtext), [Open-VSX](https://github.com/eclipse/openvsx) and many other projects has been open source. In fact, Gitpod ~~is~~ was our only closed-source project and it is a relief to change that going forward.

Reviewer: I'd remove the ~~is~~ as it was disturbing my reader flow unnecessarily (I had to think)

Reviewer 2: I think "We want to allow" doesn't sound determined enough and that's the second sentence.
Better "This allows the community to participate in the development of Gitpod, provides more transparence and makes it easier..."

<div class="wrapper">
    <a href= "https://us02web.zoom.us/webinar/register/5415977565541/WN_amoa6lnEQniLykXUYCDyBQ" target="_blank">
        <button class="button">How Gitpod uses Gitpod</button>
    </a>
</div>

# Gitpod Self-Hosted is Free

Starting today, Gitpod Self-Hosted is free for unlimited users. Organizations using Gitpod can upgrade to Gitpod Professional to get additional features: 

* [Snapshots](https://www.gitpod.io/features/#snapshot) (share a reproducible workspace with your team)
* [Live Share](https://www.gitpod.io/features/#share) (invite others into your running workspace)
* [Unlimited Prebuilds](https://www.gitpod.io/features/#prebuilt) (making ephemeral dev environments possible)
* Support (2h response during working hours)

The success of Gitpod Professional makes it possible for us to keep working towards building a new category of developer tools, which completes modern DevOps pipelines. In the future we will add additional functionality to both the open source code as well our paid plans.

**We promise that we will never remove any features from the open source code base and make it a paid feature. Everything that is open source stays open source.**

> # Getting Started

Reviewer: Can we remove this paragraph and incorporate the sentence about SaaS with the paragraph above? 

And maybe make "Install Gitpod Self-Hosted" a CTA button pointing to the install section on the website.

---

> date: Tue, 25 Aug 2020 17:00:00 UTC

Reviewer 3: Isn't the announcement time at 15:00 Germany/France time? (That would be 13:00 UTC.)

```suggestion
date: Tue, 25 Aug 2020 13:00:00 UTC
```

Author: Yeah doesn't really matter, right ¯\_(ツ)_/¯

Reviewer 3: Details are important 😇

author: svenefftinge, JohannesLandgraf
subtitle:
title: Gitpod is now Open Source 🎉
image: /Celebration-blog-post-06.jpg
teaserImage: /Celebration-blog-post-06.jpg
---

> As of today Gitpod is open source under the AGPL license at [github.com/gitpod-io/gitpod](https://github.com/gitpod-io/gitpod). This allows the community to participate in the development of Gitpod, provide more transparency and make it even easier for developers to use and integrate Gitpod in their workflows.

Reviewer 3: ```suggestion
As of today Gitpod is open source under the AGPL license at [github.com/gitpod-io/gitpod](https://github.com/gitpod-io/gitpod). This allows the community to participate in the development of Gitpod, provides more transparency and makes it even easier for developers to use and integrate Gitpod in their workflows.
```

> As of today Gitpod is open source under the AGPL license at [github.com/gitpod-io/core](https://github.com/gitpod-io/core). We want to allow the community to participate in the development of Gitpod, provide more transparency and make it even easier for developers to use and integrate Gitpod in their workflows.

Reviewer 3: The repo is `gitpod-io/gitpod`, not `gitpod-io/core`, right?

> For those of you who know us, this probably does not come as a big surprise. Working in open source is in our DNA and everything we’ve created over the past 10 years, including [Theia](https://github.com/eclipse-theia/theia), [Xtext](https://github.com/eclipse/xtext), [Open-VSX](https://github.com/eclipse/openvsx) and many other projects has been open source. In fact, Gitpod was our only closed-source project and it is a relief to change that going forward.

Reviewer 4: it should be plural, right? "have been open source"

Reviewer 3: Maybe "a big relief"?

Reviewer 3: Also it's `Open VSX` and not `Open-VSX` (we should know our own product names 😝). 

Here is a grouped, commit-in-one-click suggestion for convenience:

```suggestion
For those of you who know us, this probably does not come as a big surprise. Working in open source is in our DNA and everything we’ve created over the past 10 years, including [Theia](https://github.com/eclipse-theia/theia), [Xtext](https://github.com/eclipse/xtext), [Open VSX](https://github.com/eclipse/openvsx) and many other projects have been open source. In fact, Gitpod was our only closed-source project and it is a big relief to change that going forward.
```

> For those of you who know us, this probably does not come as a big surprise. Working in open source is in our DNA and everything we’ve created over the past 10 years, including [Theia](https://github.com/eclipse-theia/theia), [Xtext](https://github.com/eclipse/xtext), [Open-VSX](https://github.com/eclipse/openvsx) and many other projects have been open source. In fact, Gitpod was our only closed-source project and it is a relief to change that going forward.

Reviewer 3: ```suggestion
For those of you who know us, this probably does not come as a big surprise. Working in open source is in our DNA and everything we’ve created over the past 10 years, including [Theia](https://github.com/eclipse-theia/theia), [Xtext](https://github.com/eclipse/xtext), [Open VSX](https://github.com/eclipse/openvsx) and many other projects have been open source. In fact, Gitpod was our only closed-source project and it is a big relief to change that going forward.
```

> For those of you who know us,this probably does not come as a big surprise. Working in open source is in our DNA and everything we’ve created over the past 10 years, including [Theia](https://github.com/eclipse-theia/theia), [Xtext](https://github.com/eclipse/xtext), [Open VSX](https://github.com/eclipse/openvsx) and many other projects have been open source. In fact, Gitpod was our only closed-source project and it is a relief to change that going forward.

Reviewer 3: Add missing space:

```suggestion
For those of you who know us, this probably does not come as a big surprise. Working in open source is in our DNA and everything we’ve created over the past 10 years, including [Theia](https://github.com/eclipse-theia/theia), [Xtext](https://github.com/eclipse/xtext), [Open VSX](https://github.com/eclipse/openvsx) and many other projects have been open source. In fact, Gitpod was our only closed-source project and it is a relief to change that going forward.
```

> # Open Source Contributions

Reviewer 3: "Open-Source Contributions"? (With dash)

> Contributing to Gitpod should be easy and accessible for everyone. All contributions are welcome, including PRs, issues, new docs as well as updates and tweaks, blog posts, workshops, and more. Please head over to [Github](https://github.com/gitpod-io/core) to find out about the various ways you can contribute and join our [Gitpod Community](https://community.gitpod.io/).

Reviewer 3: "PRs" --> "Pull Requests"
"new docs" --> "documentation" (improving existing docs is cool too)

Also I'm not sure how I can contribute a "workshop" to Gitpod's source code.

> Contributing to Gitpod should be easy and accessible for everyone. All contributions are welcome, including pull requests, issues, documentation as well as updates and tweaks, blog posts, tutoials, and more. Please head over to [Github](https://github.com/gitpod-io/gitpod) to find out about the various ways you can contribute and join our [Gitpod Community](https://community.gitpod.io/).

Reviewer 3: "updates and tweaks" doesn't sound good. Alternatively, "typo fixes" or "general improvements" or "small fixes", something like that (one element, not two).

Also, it's GitHub, not Github. And a helpful comma.

```suggestion
Contributing to Gitpod should be easy and accessible for everyone. All contributions are welcome, including pull requests, issues, documentation as well as typo fixes, blog posts, tutoials, and more. Please head over to [GitHub](https://github.com/gitpod-io/gitpod) to find out about the various ways you can contribute, and join our [Gitpod Community](https://community.gitpod.io/).
```

Author: I think what @svenefftinge meant were not typo fixes but general tweaks etc. I will fix however, my own typo fix with Github ;)

> Over the last year, Gitpod has simplified contributions on many open source projects (see [contribute.dev](https://contribute.dev/)). Today, everybody in our team is excited to share our own development pipeline including Kubernetes preview environments, an aggressively caching build system, our slim and fast CI system and of course how Gitpod continuously beams us into ready-to-code (and debug) dev environments. [Chris](https://github.com/csweichel) gave a great talk about this setup earlier this year 👇

Reviewer 3: ```suggestion
Over the last year, Gitpod has simplified contributions to many open-source projects (see [contribute.dev](https://contribute.dev/) for examples). Today, everybody in our team is excited to share our own streamlined development pipeline including Kubernetes preview deployments, an aggressively cached build system, our own slim and fast CI system and of course Gitpod which continuously beams us into ready-to-code (and debug) dev environments. [Chris](https://github.com/csweichel) gave a great talk about this setup earlier this year 👇
```

> Over the last year, Gitpod has simplified contributions on many open source projects (see [contribute.dev](https://contribute.dev/) for examples). Today, everybody in our team is excited to share our own development pipeline including Kubernetes preview deployments, an aggressively cached build system, our slim and fast CI system and of course Gitpod, which continuously beams us into ready-to-code (and debug) dev environments. [Chris](https://github.com/csweichel) gave a great talk about this setup earlier this year 👇

Reviewer 3: "Over the last year" --> "Over the _past_ year"
"contributions on" --> "contributions _to_"
"everybody" --> "everyone"
"our own development pipeline" --> "our own _streamlined_ development pipeline"
"our slim and fast CI system" --> "our _own_ slim and fast CI system"

```suggestion
Over the past year, Gitpod has simplified contributions to many open source projects (see [contribute.dev](https://contribute.dev/) for examples). Today, everyone in our team is excited to share our own streamlined development pipeline including Kubernetes preview deployments, an aggressively cached build system, our own slim and fast CI system and of course Gitpod, which continuously beams us into ready-to-code (and debug) dev environments. [Chris](https://github.com/csweichel) gave a great talk about this setup earlier this year 👇
```

`youtube: dFMpXUsJcGM`

Naturally, we develop Gitpod in Gitpod. This allows the  whole team  to spin up fully initialized, remote dev environments on any branch at any time. 

> In line with the [pets vs. cattle](http://cloudscaling.com/blog/cloud-computing/the-history-of-pets-vs-cattle/?utm_source=thenewstack&utm_medium=website&utm_campaign=platform) analogy of the cloud-native world, we treat dev environments as automated (yet customizable) resources you can spin-up once you need them and close down (and forget about) when you are done with your task. Once you experience the peace of mind of automated, ephemeral dev environments you never want to go back.

Reviewer 3: It's "to spin up", not "to spin-up":

```suggestion
In line with the [pets vs. cattle](http://cloudscaling.com/blog/cloud-computing/the-history-of-pets-vs-cattle/?utm_source=thenewstack&utm_medium=website&utm_campaign=platform) analogy of the cloud-native world, we treat dev environments as automated (yet customizable) resources you can spin up when you need them and close down (and forget about) when you are done with your task. Once you experience the peace of mind of automated, ephemeral dev environments you never want to go back.
```

> Sven will run a webinar next Thursday, where we will showcase how we internally at Gitpod use  Gitpod and how much it has improved our workflows. Hope to see you there! 

Reviewer 3: Date is ambiguous:

```suggestion
Sven will run a webinar next week on Thursday, where we will showcase how we use Gitpod internally at Gitpod and how much it improves our workflows. Hope to see you there! 
```

<style>

> .button {

Reviewer 4: why do you not just use the same button style as elsewhere? This one is slightly different. It'd be easiest to just use the existing one, right?

Reviewer 3: Personally I like the new button style, and find it's okay to occasionally use a different style e.g. for one particular blog post (here there is the confetti theme, might as well have a few slightly different buttons for a change 🙂).

Reviewer 4: It's important to have a consistent CI (corporate identity). If we want to make changes, they should be obvious and clear to rule out the risk that they are perceived as mistakes or as the result of sloppy work. Adding a slightly different version to an existing version is rather confusing and mostly appears unprofessional.

Author: @ChristinFrohne I changed the button to the original UI (did build the thing myself before)

> Sven will run a webinar next week on Thursday, where we will showcase how we use Gitpod internally at Gitpod and how much it improves our workflows. Hope to see you there! 

Reviewer 3: "our worfklow" singular sounds better?

```suggestion
Sven will run a webinar next week on Thursday, where we will showcase how we use Gitpod internally at Gitpod and how much it improves our workflow. Hope to see you there! 
```

.button {
  background-color: #0087BE; 
  border: none;
  border-radius: 100px;
  color: white;
  padding: 12px 20px;
  text-align: center;
  text-decoration: none;
  display: inline-block;
  font-size: 14px;
  margin: 25px 1px;
  cursor: pointer;
  opacity: 1;

}

.button:hover {
	opacity: 0.85;

}
.wrapper {
    text-align: center;
}
</style>

<div class="wrapper">
    <a href= "https://us02web.zoom.us/webinar/register/5415977565541/WN_amoa6lnEQniLykXUYCDyBQ" target="_blank">
        <button class="button">How Gitpod uses Gitpod</button>
    </a>
</div>

> # Gitpod Self-Hosted is Free

Reviewer 3: Small clarification?

```suggestion
# Gitpod Self-Hosted is Now Free
```

> The [SaaS version of gitpod.io](https://www.gitpod.io/pricing/#) remains the most convenient way to streamline your development workflows with continuously prebuilt dev environments. 

Reviewer 3: Clarify, don't make self-hosted sound bad:

```suggestion
The SaaS version of Gitpod, [gitpod.io](https://www.gitpod.io/pricing/) remains the easiest way to streamline your development workflows with continuously prebuilt dev environments. 
```

> The [SaaS version of gitpod.io](https://www.gitpod.io/pricing/#) remains the easiest way to streamline your development workflows with continuously prebuilt dev environments. 

Reviewer 3: Clarify:

```suggestion
The [SaaS offering of gitpod.io](https://www.gitpod.io/pricing/#) remains the easiest way to streamline your development workflows with continuously prebuilt dev environments. 
```

> The [SaaS version of gitpod.io](https://www.gitpod.io/pricing/#) remains the most convenient way to streamline your developer workflows with continuously prebuilt dev environments. 

Reviewer 4: "remains the most convenient way" sounds like Gitpod Self-Hosted is not a great choice. Maybe rather put it in a way that both options sound great?

Reviewer 3: Great points. Made a suggestion to address this, so resolving here.

> In case you want to host Gitpod on your own infrastructure or private cloud, starting today, Gitpod Self-Hosted is free for unlimited users. Organizations using Gitpod can upgrade to Gitpod Professional to get additional features: 

Reviewer 3: Clarify

```suggestion
But if you want to host Gitpod on your own infrastructure or private cloud, starting today, Gitpod Self-Hosted is free for unlimited users. Also, organizations using Gitpod Self-Hosted can purchase an enterprise license to get additional features: 
```

> In case you want to host Gitpod on your own infrastructure or private cloud, starting today, Gitpod Self-Hosted is free for unlimited users. Organizations using Gitpod Self-Hosted can purchase an enterprise licence in order to get additional features: 

Reviewer 3: "licence" --> "license"
"additional features _like_"

```suggestion
In case you want to host Gitpod on your own infrastructure or private cloud, starting today, Gitpod Self-Hosted is free for unlimited users. Organizations using Gitpod Self-Hosted can purchase an enterprise license in order to get additional features like:
```

* [Snapshots](https://www.gitpod.io/features/#snapshot) (share a reproducible workspace with your team)

> * [Live Share](https://www.gitpod.io/features/#share) (invite others into your running workspace)

Reviewer 3: Note: I think "Live Share" is a Microsoft term. But maybe that's okay.

* [Unlimited Prebuilds](https://www.gitpod.io/features/#prebuilt) (making ephemeral dev environments possible)
* Support (2h response during working hours)

> 

Reviewer 4: Ah quite annoying that this information is never aligned. On the website it says nothing about extra support, but it mentions "Admin Dashboard". Might be okay to have a slight difference here, but we need to improve the internal flow

Reviewer: Yeah, let's replace the Support with 'Admin Dashboard' for now.

> Offering a paid plan for enterprises makes it possible for us to keep working towards building a new category of developer tools, which completes modern DevOps pipelines. In the future we will add additional functionality to both the open source code as well our paid plans.

Reviewer 3: We're not building a bunch of random tool, we're building one brand new tool:

```suggestion
Offering a paid plan for enterprises makes it possible for us to keep working towards building a brand new type of developer tool, which completes modern DevOps pipelines. In the future we will add additional functionality to both the open source code as well our paid plans.
```

Author: It is about that we are spearheading a new category in dev tooling around automated dev environments. Will change to "new category in developer tooling, which completes modern DevOps pipelines"

> The success of Gitpod Professional makes it possible for us to keep working towards building a new category of developer tools, which completes modern DevOps pipelines. In the future we will add additional functionality to both the open source code as well our paid plans.

Reviewer 4: I'd ask myself immediately "what kind of success" ? maybe rather "Offering a paid plan for enterprises makes it possible to.. "
